### DISCUSSION TITLE
NoneBot Flutter GUI

### DISCUSSION THREAD

**Discussion by XTxiaoting14332** [nonebot/discussions#473] [应用]
闲的没事干写的 来点用户吧😭😭😭
https://github.com/NoneBotGUI/nonebot-flutter-gui/

网站https://nbgui.none.bot/

求求了😭👊🏻

**Comment by verydemo** [comment]
后续支持docker部署，浏览器打开界面么

**Reply by XTxiaoting14332** [reply]
webui暂时无法实现，因为依赖了系统命令，如果要webui则需要依靠http通信，但我还没学（
docker部署后期应该可以支持，但是得等我高考完才能做了qwp

**Reply by XTxiaoting14332** [reply]
> 后续支持docker部署，浏览器打开界面么

已支持 webui 
项目地址： https://webui.nbgui.top
可使用 Docker 部署 webui
